api/graphql/schema.graphql

```diff
@@ -287,6 +287,8 @@ enum PoolCandidateStatus {
     UNAVAILABLE
     EXPIRED
     PLACED_CASUAL
+    DRAFT
```

Comment:
I thought we were just going to add a DRAFT status and whether or not it is expired would be programmatically calculated base on dates?

Reply:
no I don't think so, #3660 seems to require the addition of DRAFT_EXPIRED as a status too
I've just included the draft statuses for the sake of submitted_at logic, the stuff to handle statuses will come in that issue

Reply:
Eugh, you're right but I don't like it.  :smile: We can remove the status later, I guess, if we need to.